src/components/URLImportDialog/index.js

```diff
@@ -102,19 +111,47 @@ const validURL = (possibleURL) => {
     }
 };
 
+/**
+ * Takes a URL string and returns just the file name without any query
+ * params or tags.  This is how the DE's import job will name the file
+ * as well.
+ * @param urlString
+ * @returns {string}
+ */
+const urlFileName = (urlString) =>
+    urlString.split("/").pop().split("#")[0].split("?")[0];
+
 const URLImportTextField = (props) => {
     const classes = useStyles();
     const [uploadURL, setUploadURL] = useState("");
     const [isValidURL, setIsValidURL] = useState(false);
     const [hasFirstFocused, setHasFirstFocused] = useState(false);
 
-    const { addURLFn, intl } = props;
+    const { path, intl } = props;
+
+    const [importUrl] = useMutation(uploadByUrl, {
+        onError: (error) => {
+            const errorPath = getErrorData(error)?.path;
+            const duplicateName = urlFileName(errorPath);
+            const text =
+                getErrorCode(error) === ERROR_CODES.ERR_EXISTS
+                    ? fmt(intl, "fileExists", {
+                          name: duplicateName,
+                          path: path,
+                      })
+                    : fmt(intl, "fileImportFail");
+            announce({
+                text,
+                variant: AnnouncerConstants.ERROR,
+            });
```

Comment:
There is an option to provide `Custom` action in announcer. In this case, it will be `View Details` which will pop open a `ErrorDialog` with details. Please see - https://github.com/cyverse-de/sonora/blob/71e135e2b7e19451dd3e09011f39f47dcc219a48/src/components/data/listing/Listing.js#L127